wagtail/admin/templates/wagtailadmin/shared/panel.html

```diff
@@ -0,0 +1,62 @@
+{% load wagtailadmin_tags i18n %}
+{% comment %}
+    Variables this template accepts:
+
+    id_prefix - A prefix for all id attributes.
+    classes - List of CSS classes to use for the panel.
+    class_name - String of CSS classes to use for the panel.
+    id - Unique to the page.
```

Comment:
should this be `panel_id` (or should the code be updated to use `id`?)

Reply:
The code does use `id` on line `17`.